lstchain/io/lstcontainers.py

```diff
@@ -31,18 +32,22 @@ class DL1ParametersContainer(Container):
     TODO: maybe fields could be inherited from ctapipe containers definition
         For now I have not found an elegant way to do so
     """
-    intensity = Field(None, 'total intensity (size)')
-    log_intensity = Field(None, 'log of total intensity (size)')
-
-    x = Field(None, 'centroid x coordinate', unit=u.m)
-    y = Field(None, 'centroid x coordinate', unit=u.m)
-    r = Field(None, 'radial coordinate of centroid', unit=u.m)
-    phi = Field(None, 'polar coordinate of centroid', unit=u.rad)
-    length = Field(None, 'RMS spread along the major-axis', unit=u.deg)
-    width = Field(None, 'RMS spread along the minor-axis', unit=u.deg)
-    psi = Field(None, 'rotation angle of ellipse', unit=u.rad)
-    skewness = Field(None, 'measure of the asymmetry')
-    kurtosis = Field(None, 'measure of the tailedness')
+    intensity = Field(np.float64(np.nan), 'total intensity (size)')
+    log_intensity = Field(np.float64(np.nan), 'log of total intensity (size)')
+
+    x = Field(u.Quantity(np.nan, u.m), 'centroid x coordinate', unit=u.m)
+    y = Field(u.Quantity(np.nan, u.m), 'centroid x coordinate', unit=u.m)
+    r = Field(u.Quantity(np.nan, u.m), 'radial coordinate of centroid', unit=u.m)
+    phi = Field(Angle(np.nan, u.rad), 'polar coordinate of centroid',
+                unit=u.rad)
+    length = Field(u.Quantity(np.nan, u.deg), 'RMS spread along the major-axis',
+                   unit=u.deg)
+    width = Field(u.Quantity(np.nan, u.deg), 'RMS spread along the minor-axis',
+                             unit=u.deg)
+    psi = Field(Angle(np.nan, u.rad), 'rotation angle of ellipse', unit=u.rad)
+
+    skewness = Field(np.nan, 'measure of the asymmetry')
+    kurtosis = Field(np.nan, 'measure of the tailedness')
     disp_norm = Field(None, 'disp_norm [m]', unit=u.m)
```

Comment:
disp values should also be `np.nan * u.m`

Reply:
Fine, but values which are not calculated in the dl1 step are better left as None and then they do not go to the output. Here I want to do minimal changes for this to work (because there are always unwanted side effects of changes). As you know this DL1 container is anyway deprecated.